censys/asm/api.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, api_key: Optional[str] = None, **kwargs):
         )
 
         if not self._api_key:
-            raise CensysMissingApiKeyException("No ASM API key configured.")
+            raise CensysException("No ASM API key configured.")
```

Comment:
We talked about this in meeting, but for the sake of it being in the review, this will cause tests to fail if `CENSYS_ASM_API_KEY` does not exist in the test environment.

This is problematic for development since a functioning key isn't actually necessary for the tests to run (calls are mocked)

Reply:
This could also potentially be problematic since running the tests will decrement the user's API quota (though that probably isn't a huge concern)

Reply:
Fixed in f101d11